.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+
+on:
+  push:
+    paths:
+      - '.github/workflows/test.yml'
+      - 'src/*'
+      - 'profiling/*'
+      - 'benchmarks/*'
+
+jobs:
+  test_cpython:
+    name: CPython ${{ matrix.py }} for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
+        py: [3.5, 3.6, 3.7]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Install Rust
+        run: |
+          wget --https-only --secure-protocol=TLSv1_2 -O rustup-init.sh https://sh.rustup.rs
+          sh ./rustup-init.sh --default-toolchain nightly -y -v
+          echo ::add-path::$HOME/.cargo/bin
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.py }}
+          architecture: x64
+      - name: Install Pipenv
+        run: |
+          pip3 install --user pipenv
+          echo ::add-path::$HOME/.local/bin
+      - name: Setup Virtualenv
+        run: |
+          # squelch the warning
```

Comment:
Could we add the warning text here for future reference?

Reply:
I think now that I don't have to keep debugging that step that it's a fine choice :)